pkg/apiserver/metrics/metrics.go

```diff
@@ -42,7 +42,7 @@ var (
 			// Use buckets ranging from 125 ms to 8 seconds.
 			Buckets: prometheus.ExponentialBuckets(125000, 2.0, 7),
 		},
-		[]string{"verb", "resource"},
+		[]string{"verb", "resource", "client"},
```

Comment:
FYI: I think in all our tests we are looking more into "apiserver_request_latencies_summary" (which is the metric below) - it is much more useful.
We need to check if we are not using this metric programatically anywhere, but I think this is fine.